.gitlab-ci.yml

```diff
@@ -44,7 +44,7 @@ build:
       -t $BUILDER_TAG
       -t $IMAGE_TAG
       --file $CI_PROJECT_DIR/Dockerfile
-      --cache-from $BUILDER_TAG
+      --no-cache
```

Comment:
Nice! I also recommend that you add `--pull` to your `docker build` command. This will ensure that any updates to the base docker image get reflected in your built container.

Reply:
nice!